### PR TITLE
feat(node): use openssl ca certificates

### LINKED ISSUE #61: Automatically configure Node.js for custom certificates
If custom certificates are provided, we should automatically configure Node.js to use them. By default they use their own bundled CA certs, so I think would ignore any custom certs added to Linux.

Should we configure `--use-openssl-ca` or `SSL_CERT_DIR` or `SSL_CERT_FILE`?

### PR DESCRIPTION
**Pull request by viceice**
closes #61

### CODE REVIEW

File: docs/custom-root-ca.md [hunk 1]
```diff
@@ -18,12 +18,11 @@ RUN update-ca-certificates
 ## Runtime install
 
 Most OpenSSL base tools (and maybe BoringSSL) support `SSL_CERT_FILE` environment for additional custom root ca files.
-For NodeJS `NODE_EXTRA_CA_CERTS` is required.
+We use `NODE_OPTIONS="--use-openssl-ca"`, so NodeJS is using the same certifacte options as OpenSSL.
```
Comment (rarkins) [inline docs/custom-root-ca.md:21]: Typo

**Review by rarkins — CHANGES_REQUESTED** [review]

**Comment by rarkins** [comment]
Any easy way to verify it works?

File: docs/custom-root-ca.md (hunk 1 above)
Comment (viceice) [inline docs/custom-root-ca.md:21]:
```suggestion
We use `NODE_OPTIONS="--use-openssl-ca"`, so NodeJS is using the same certificate options as OpenSSL.
```

File: test/latest/Dockerfile [hunk 2]
```diff
@@ -77,7 +77,7 @@ ARG APT_HTTP_PROXY
 RUN install-tool node 14.17.3
 
 RUN set -ex; \
-  nginx && su -c 'NODE_EXTRA_CA_CERTS=/test/ca.pem node request.mjs' ${USER_NAME}
+  nginx && su -c 'SSL_CERT_FILE=/test/ca.pem node request.mjs' ${USER_NAME}
```
Comment (viceice) [inline test/latest/Dockerfile:80]: This is the test. Simple https request with a custom root certificate.

**Review by rarkins — APPROVED** [review]

**Pull request merged**